### ISSUE TITLE
Helm Standalone segfaults when routed through JACK

### ISSUE THREAD

**Issue by Charles-the-Cat** [mtytel/helm#227]
I'm using AVLinux 2019.4.10 (based on Debian 9.8, kernel 4.16.12-rt5-avl1, 64-bit),, JACK version "jackdmp version 1.9.12 tmpdir /dev/shm protocol 8", and Helm Standalone v0.9.0.
I want to route audio through Calf Studio Gear, so I switch Helm's 'Audio device type' to JACK and choose 'Calf Studio Gear' as output. The appropriate plugin inputs show up under 'Active output channels', but clicking them does nothing.
From here I try connecting from Calf itself. Its 'Available JACK Audio Outputs' include 'Helm:out_1', 'Helm:out_2', and 'Helm:out_3' (sometimes higher numbers, too), but upon selecting any of them Helm segfaults without explanation.

**Comment by JRSV** [comment]
I can confirm this in Ubuntu 19.10. any news on this? In my case, I am trying to route helm to Reaper. When trying from the pref menu in helm, it doesn't connect and when trying to do it in Qjackctl helm crashes.

**Comment by trebmuh** [comment]
Maybe the same than #124

**Comment by JRSV** [comment]
yup.. seems its a long standing problem, makes it hard to perform with all of JACK{s flexibilities.

**Comment by trebmuh** [comment]
agreed
